### PR TITLE
set up workflow object in config and relevant changes in rvbtn function

### PR DESCRIPTION
**Pull request by MottMac-PH**

### CODE REVIEW

File: SiteAssets/pages/3.0/scripts/PROCESS.CONTROLLER.js [hunk 1]
```diff
@@ -4575,11 +4575,11 @@ function hazardreviewbuttonaction() {
                                             }).done(r => {
                                                 let item = ((r.d.results[0].cdmRAMS != null));
                                                 if (item) {
-                                                    tdata.push("cdmCurrentStatus|Under Construction Manager review");
+                                                    tdata.push(`"cdmCurrentStatus|${configData["Workflow"][a]["nextWorkFlowState1"]}`); //Editable workflow config. Patrick Hsu, 30 Jan 2024
```
Comment (JHolland98) [inline SiteAssets/pages/3.0/scripts/PROCESS.CONTROLLER.js]: The double quote mark after the special quote character needs to be removed

Comment (JHolland98) [reply SiteAssets/pages/3.0/scripts/PROCESS.CONTROLLER.js]: This is present in a number of other places also

File: SiteAssets/pages/3.0/scripts/configData.js [hunk 2]
```diff
@@ -20,5 +19,34 @@ configData = {
...
+    'Exportable workflow states': ['Requires mitigation', 'Assessment in progress', 'Under peer review', 'Under design manager review'],
+    
+    //Editable workflow config section. Patrick Hsu, 29 Jan 2024
+    'Workflow' : { 
+        'initiatereview' : { 
+            'nextWorkFlowState': 'Under peer review', 
+            'userRoles': ['Designer','Construction Engineer','Design Manager','Construction Manager','Principal Designer'] // who can review/edit the hazard at this point
+        },
+        'peerreview' : { 
+            'nextWorkFlowState1': 'Under Construction Manager review',
```
Comment (JHolland98) [inline SiteAssets/pages/3.0/scripts/configData.js]: I think there should only be one nextWorkFlowState here and it should be 'Under design manager review'

File: SiteAssets/pages/3.0/scripts/PROCESS.CONTROLLER.js (hunk 1 above)
Comment (JHolland98) [reply SiteAssets/pages/3.0/scripts/PROCESS.CONTROLLER.js]: Also lets just leave the workflow for RAMS hazard as it is

File: SiteAssets/pages/3.0/scripts/configData.js [hunk 3]
```diff
@@ -5,7 +5,6 @@ configData = {
     'Client Review': false, // Does the project require a client review to be included in the workflow?
     'Client Name': 'Client', // What is the name of the client of the project?
     'Archive hazards permissions': ['System admin'], // Which user roles should be authorised to archive hazards. This is given as a list. E.g., ['System admin', 'Designer', 'Construction Manager'].
-    'Sync Client Hazard Permissions': ['System admin'], // Defines who can use 'Sync Client Hazards' button.
```
Comment (JHolland98) [inline SiteAssets/pages/3.0/scripts/configData.js:8]: We need this line to be added back into the config file

**Review by JHolland98 — CHANGES_REQUESTED** [review]

File: SiteAssets/pages/3.0/scripts/configData.js [hunk 4]
```diff
@@ -22,15 +22,14 @@ configData = {
     'Exportable workflow states': ['Requires mitigation', 'Assessment in progress', 'Under peer review', 'Under design manager review'],
     
     //Editable workflow config section. Patrick Hsu, 29 Jan 2024
-    'Workflow' : { 
+    /*'Workflow' : {
```
Comment (JHolland98) [inline SiteAssets/pages/3.0/scripts/configData.js:25]: Lets remove this commented out workflow object

File: SiteAssets/pages/3.0/scripts/PROCESS.CONTROLLER.js (hunk 1 above)
Comment (MottMac-PH) [reply SiteAssets/pages/3.0/scripts/PROCESS.CONTROLLER.js]: I have checked and removed all the places with double quote marks after special quote

File: SiteAssets/pages/3.0/scripts/configData.js (hunk 3 above)
Comment (MottMac-PH) [reply SiteAssets/pages/3.0/scripts/configData.js:8]: I've added it back

**Comment by MottMac-PH** [comment]
Hi James, I've made all the changes and re-committed. Please review thanks!

**Pull request merged**